util/nrt_diff/nrtDiff.py

```diff
@@ -17,6 +17,7 @@
 import filecmp
 import imp
 import itertools
+from xml.dom import minidom
```

Comment:
Why don't we use lxml for other XML as well ?

Reply:
it will be done in following PRs